client/src/components/SearchBar.css

```diff
@@ -0,0 +1,53 @@
+#searchBar{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#form {
```

Comment:
Since this is all in the same class, I think it's best practice to start with . instead of #, so it's organized as the components in the searchBar id. (# is for id, and . is for className)